pages/dashboards/templates/dashboards/recovac.html

```diff
@@ -0,0 +1,917 @@
+                    2021, whilst data on ICU admissions resulting from COVID-19 infection are
+                    available from March 2020.
+                    Further, the latest date for which data is available can differ between the
+                    two datasets.
+                    The default view will show all of the data available for both
+                    datasets.
+                    However, it is useful for align the timeframes, especially when
+                    making inferences about the effects of vaccination on ICU admission.
+                    Use the 'Align timeframe' button in the 'Timeframe' buttons list to align
+                    the timeframes.
+                    Click the 'Show all data' button to view all of the available data.
+                </p>
+
+                <h6 class="text-base font-semibold text-pp-dark-blue">
+                    Accurately read data values
+                </h6>
+                <p>
+                    It is possible to view the underlying data values by hovering the cursor
+                    over the graph.
+                    All values for a given date are shown together.
+                </p>
+
+                <h6 class="text-base font-semibold text-pp-dark-blue">
+                    Other features
+                </h6>
+                <p>
+                    When hovering over the plot with the cursor, additional grey icons appear in
+                    the top right.
+                    The +, -, and magnifying glass icons can be used to zoom in/out of the plot.
+                    Alternatively, it is possible to zoom into a given part of the graph by
+                    clicking and dragging with the cursor to select that portion.
+                    The autoscale and reset axes icons (which look like a box containing arrows
+                    and a house, respectively) can be used to scale the axes appropriately for
+                    the data selected.
+                    The plot can be downloaded in .png format by clicking on the camera icon.
+                </p>
+            </div>
+        </details>
+
+        <div class="md:hidden bg-pp-pale-grey/30 border-l-4 border-pp-mid-blue text-pp-dark-blue p-4" role="status">
+            <p class="m-0">Rotating your phone may improve graph layout</p>
+        </div>
+
+        <div class="mb-3 max-w-full overflow-x-auto">
+            <div>
+                {% if swedishpop_subplot_button %}
+                    {{ swedishpop_subplot_button|safe }}
+                {% else %}
+                    {% include "dashboards/partials/plot_loading_error.html" %}
+                {% endif %}
+            </div>
+        </div>
+
+        <p>
+            <strong>Code used to produce plots:</strong>
+            <a
+                href="https://github.com/ScilifelabDataCentre/pathogens-portal-visualisations/blob/main/RECOVAC/Swedishpop_vaccinecov_dataprep.py"
```

Comment:
as mentioned above, this and the subsequent URLs will also need to be changed. So we should make a note of it.